test/extensions/common/aws/BUILD

```diff
@@ -38,6 +39,11 @@ envoy_cc_test(
 envoy_cc_test(
     name = "utility_test",
     srcs = ["utility_test.cc"],
+    copts = envoy_select_boringssl(
+        [
+            "-DENVOY_SSL_FIPS",
+        ],
+    ),
```

Comment:
Configurable would avoid this also?

Reply:
Another reason is that for configuration to know whether to enable or disable FIPS mode, the static bootstrap generator has to get the `envoy --version` to distinguish between BoringSSL vs BoringSSL-FIPS on the cli output. So I thought to keep the logic with Envoy.